tests/communities/test_communities.py

```diff
@@ -17,7 +17,7 @@
 
 @allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703084', 'Create community')
 @pytest.mark.case(703084)
-# @pytest.mark.skip(reason="https://github.com/status-im/desktop-qa-automation/issues/167")
+@pytest.mark.skip(reason="https://github.com/status-im/desktop-qa-automation/issues/167")
```

Comment:
So this test works even without Vlad's changes from here https://github.com/status-im/desktop-qa-automation/pull/327?

Reply:
Ah, sorry. Yes, of course, let's do this